stories/file-download/file-download.stories.tsx

```diff
@@ -52,7 +52,7 @@ export const Default: StoryObj<Component> = {
 
         return (
             <FileDownload
-                fileItems={fileItems}
+                fileItems={[fileItems[2]]}
```

Comment:
is this change intended? as it's for a separate story

Reply:
oh shit, copy pasta lol